rpc/proto/mayastor.proto

```diff
@@ -194,6 +195,7 @@ message DestroyNexusRequest   {
 message AddChildNexusRequest {
   string uuid = 1;    // uuid of the nexus
   string uri = 2;     // URI of the child device to be added
+  bool rebuild = 3;   // auto start rebuilding
```

Comment:
Yeah I had no_rebuild initially which is when I hit that "_" issue in JS so I thought it would be better to avoid it. I did not expose the rebuild flag via the mayastor-client app but I do agree that the negation is better so I'll make the change (this time no "_" in the name and should just work)

Reply:
thanks